community/kernel/src/main/java/org/neo4j/kernel/impl/store/record/RelationshipRecord.java

```diff
@@ -182,4 +182,22 @@ public void setIdTo( PropertyRecord property )
     {
         property.setRelId( getId() );
     }
-}
\ No newline at end of file
+
+    @Override
+    public RelationshipRecord clone()
```

Comment:
Note when merging to 3.0, 3.0 already has a clone method so this should be scrapped in favor of whatever is already there.